db/db_agency.go

```diff
@@ -0,0 +1,80 @@
+package structs
```

Comment:
Sugestão: ter um pacote novo chamado storage, que toma conta tanto da parte de armazenamento no BD e no serviço de armazenamento. O motivo da sugestão é que essas coisas são super entrelaçadas e talvez não faça sentido separá-las de início.

Esse pacote db ser apagado ao final do MVP.

Reply:
OK!